docs/reference/special-theme-elements.md

```diff
@@ -271,6 +270,17 @@ alt: My figure text
 This figure should be entirely in the margin.
 ```
 
+## Sidenotes and marginnotes
+
+Here's a sentence[^sn1] with multiple [^sn2] sidenotes.
+
+[^sn1]: Test sidenote 1.
+[^sn2]: Test sidenote 2.
+
+Here's a sentence[^mn1] with multiple marginnotes[^mn2].
+
+[^mn1]: {-} Test marginnote 1.
+[^mn2]: {-} Test marginnote 2.
 
 ## Nested admonitions
```

Comment:
```suggestion
[^mn1]: {-} Test marginnote 1.
[^mn2]: {-} Test marginnote 2.


Sidenotes inside of admonitions should behave the same:

:::{note}
An admonition with a sidenote defined in the admonition[^snam1] and another defined outside of the admonition [^snam2].

[^snam1]: Sidenote defined in the admonition.

:::

[^snam2]: Sidenote defined outside the admonition.


## Nested admonitions
```

Reply:
Here's the text I tried to test the buggy behavior when you define sidenotes in an admonition